src/analyser_test.cpp

```diff
@@ -20,17 +20,20 @@ int run(std::string filename) {
 
   for (auto r : recs) {
     std::cout << r << std::endl;
-    break;
+
   }
 
   auto states = parser.get_states();
   for (auto s : states) {
     std::cout << s << std::endl;
   }
 
-  auto domain_stats = analyser::DomainStat<log_properties::FILE>(recs);
+  auto runner_cpp_stats = analyser::DomainStat<log_properties::FILE>(recs);
   // domain_stats.print_domain_stats();
-  std::cout << domain_stats << std::endl;
+  std::cout << runner_cpp_stats << std::endl;
+
+  auto thread_stat = analyser::DomainStat<log_properties::THREAD>(recs);
```

Comment:
did not know that log_properties::THREAD works, i was going with clayer::THREAD, let's be consistent with one.